Configuration/XMLUtil.py

```diff
@@ -101,20 +101,28 @@ def SetConnection(self, beboardid, uri, address_table):
   def SetRegisterValue(self, RegisterSettings):
     self.RegisterSettings = RegisterSettings
 
+
+
+#the newer version of code are being usd fcboardC.py in XMLgenerator
+"""
```

Comment:
If the code is not being used anymore it is best to delete it, not comment it out. Github will keep track of previous versions for us.

Reply:
codes are removed